### ISSUE TITLE
Monsters Spawn in Mars City Underground

### ISSUE THREAD

**Issue by macweirdo42** [KozGit/DOOM-3-BFG-VR#350]
Hey, I'm having an issue where monsters spawn the moment Mars City Underground loads up.  In other words, I'm waiting for the guy to unlock the case to grab my pistol, and there's already a z-sec zombie shooting me!  Is there a fix for this?

**Comment by NPi2Loup** [comment]
I don't think this is a technical issue. 
The guy you see isn't a zombie.
Have you try to start to this map ? (with dev menu)

I just try => no problem

**Comment by macweirdo42** [comment]
Yeah, he is a zombie, he starts shooting me.  I'm not talking about the security guy behind the glass that opens the weapons case for you.  He's there, chatting like normal, and this zombie with a pistol just starts blasting me while I'm waiting to get my gun.  It's the guy that would normally be the last enemy of the level after you see the hell wave and work your way back to the base.  Then as I try to go forward, other zombies and imps keep appearing, all while the civilians are still around just following their scripts.

No, I haven't tried starting directly to that map.

**Comment by NPi2Loup** [comment]
Never seen this bug. It's seems there is some context var disorder. Your save may be corrupt, but since its a very early map, it strange.
Are you sure you use the cleaner install as possible ? no mods ? 
Which difficulty level ?

You should try to start directly to that map (use Dev choice in game menu)

**Comment by guy1101011** [comment]
Is it a legit steam version?
